clang/lib/AST/ByteCode/InterpBuiltin.cpp

```diff
@@ -2736,12 +2736,14 @@ static bool interp__builtin_ia32_pmul(InterpState &S, CodePtr OpPC,
   return true;
 }
 
-static bool interp__builtin_elementwise_fma(InterpState &S, CodePtr OpPC,
-                                            const CallExpr *Call) {
+static bool interp__builtin_elementwise_triop_fp(
+    InterpState &S, CodePtr OpPC, const CallExpr *Call,
+    llvm::function_ref<APFloat(const APFloat &, const APFloat &,
+                               const APFloat &, const FPOptions &)>
```

Comment:
We might end up just always passing the rounding mode instead of the `FPOptions`, I'm not sure what else we need it for. Otherwise, we should pass them as a copy since they're only 4 bytes and other code does copies as well.